src/xtgeo/xyz/_xyz_roxapi.py

```diff
@@ -440,7 +440,8 @@ def _roxapi_export_xyz(
 
     df = _apply_pfilter_to_dataframe(df, pfilter)
     if df.empty:
-        raise ValueError("Empty dataframe, no data left after filtering")
+        logger.warning("Empty dataframe after filtering! Skipping object update...")
```

Comment:
Some lines up, a `logger.warning` and return is done when the dataframe is empty. Here, after filtering, raising a `ValueError` is quite odd and inconsistent (blame me...), so I change behavior here. Having empty data after filtering is fully valid.